libs/novui/src/components/button/Button.tsx

```diff
@@ -1,22 +1,106 @@
+import { Button as ExternalButton, ButtonProps, ButtonStylesNames } from '@mantine/core';
 import { FC } from 'react';
-import { CorePropsWithChildren } from '../../types';
-import { Button as ExternalButton, ButtonProps } from '@mantine/core';
+import { colorPaletteGradient } from '../../ingredients/colorPaletteGradient.ingredient';
+import { css, cx, sva } from '../../../styled-system/css';
 import { IconType } from '../../icons';
-import { css } from '../../../styled-system/css';
+import { CorePropsWithChildren } from '../../types';
+
+type ButtonVariant = 'filled' | 'outline' | 'borderless';
 
 export interface IButtonProps
   extends CorePropsWithChildren,
     React.ButtonHTMLAttributes<HTMLButtonElement>,
-    Pick<ButtonProps, 'size' | 'variant'> {
+    Pick<ButtonProps, 'size'> {
   Icon?: IconType;
+  variant?: ButtonVariant;
 }
 
-export const Button: FC<IButtonProps> = ({ children, Icon, size = 'md', variant = 'filled', ...buttonProps }) => {
+const SLOTS: ButtonStylesNames[] = ['root', 'inner', 'label', 'loader', 'section'];
+
+const buttonRecipe = sva({
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of.  -->
🗒 **note (non-blocking):** Unfortunately layer styles with mantine are not working when I try them, so for now the `important` below are necessary. I will dig back into it further, but didn't want to get blocked on it while defining components that everyone else will likely need‏